.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```

Comment:
Can you please remove this file?

Reply:
I removed that file in my following commit. The pull request contains 4 commits. 
In my latest commit I also changed the naming from eval_x to val_x.